pkts-sip/src/main/java/io/pkts/packet/sip/impl/SipMessageBuilder.java

```diff
@@ -641,46 +652,48 @@ public T build() {
                         contentLengthHeader = finalHeader;
                     } else {
                         msgSize += finalHeader.getBufferSize() + 2;
-                        finalHeaders.add(finalHeader);
+                        finalHeaders.computeIfAbsent(finalHeader.getName().toString(), k -> new ArrayList<>()).add(finalHeader);
```

Comment:
in all the `new ArrayList()` below, should we limit the initial size from 10 to perhaps 4?